config/analytics_blocklist.yml

```diff
@@ -86,18 +95,25 @@
   - in_progress_steps
   - employment_history_section_completed
   - qualifications_section_completed
+  - place_of_worship
+  - religious_referee_name
+  - religious_referee_address
+  - religious_referee_role
+  - religious_referee_email
+  - religious_referee_phone
+  - baptism_address
+  - baptism_date
```

Comment:
Some issue with the rebasing/branching? This should already be present on the base branch instead of being introduced by this branch.